deb/common/control

```diff
@@ -7,7 +7,7 @@ Build-Depends: bash,
                ca-certificates,
                cmake,
                dh-apparmor,
-               dh-systemd,
+               debhelper (>= 9.20160709) | dh-systemd,
```

Comment:
Same comment as in https://github.com/docker/containerd-packaging/pull/212#discussion_r555897064 - I don't think we need to go to this level anymore. :smile:

```suggestion
               debhelper (>= 10~),
```

(and dropping `--with systemd` in `deb/common/rules`, which needs to be tested but should work)